.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: e67779cee2a3878c2ba9ede50f2121cf21c1e99c
+          ref: 1d77dcffdffc658e4defa6021a60ca1e9d065c61
```

Comment:
Please update the other workflows to also use the same ref. I'd like to keep them all running the same code.

https://github.com/iree-org/iree/blob/cda70e8a3e235baf50b7da31c8df75d5db3d56a9/.github/workflows/pkgci_regression_test_amdgpu_rocm.yml#L62-L63 https://github.com/iree-org/iree/blob/cda70e8a3e235baf50b7da31c8df75d5db3d56a9/.github/workflows/pkgci_regression_test_amdgpu_vulkan.yml#L59-L60 https://github.com/iree-org/iree/blob/cda70e8a3e235baf50b7da31c8df75d5db3d56a9/.github/workflows/pkgci_regression_test_nvidiagpu_cuda.yml#L59-L60 https://github.com/iree-org/iree/blob/cda70e8a3e235baf50b7da31c8df75d5db3d56a9/.github/workflows/pkgci_regression_test_nvidiagpu_vulkan.yml#L59-L60

(I just find/replace across the repo from the previous commit SHA to the new one)

Reply:
If you bump to the latest commit, you can also add `--no-skip-tests-missing-files` to the pytest flags